frontend/src/main.ts

```diff
@@ -0,0 +1,12 @@
+import { enableProdMode } from '@angular/core';
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+
+import { AppModule } from './app/app.module';
+import { environment } from './environments/environment';
+
+if (environment.metadata.environment === 'prod')
+{
+	enableProdMode();
+}
```

Comment:
after the if you add:

```
else
{
    console.table(environment.metadata);
}
```

Reply:
Throws error because the console on the linter, I check yesterday